README.md

```diff
@@ -83,6 +85,8 @@ These _soft_ real-time application greatly benefit from being able to adapt thei
   * Tweak quality-vs-speed knobs in the game’s rendering engine (shadows
     quality, texture filtering, view distance)
 
+Technically these can be accomplished by knowing thermal states (e.g., is the system being passively cooled - throttled) as well as CPU pressure states for the threads the site is using such as main thread and workers. System thermals is a global state and can be affected by other apps and sites than the observing site.
```

Comment:
"System thermals state is a global state."

I understand the grammar shortcut, but looks weird to have "thermals is "